.eslintrc.js

```diff
@@ -0,0 +1,52 @@
+module.exports = {
```

Comment:
just a thought while reviewing this, have we considered shared `eslint`/`templatelint`/`prettier`/etc definitions across repos?

Reply:
Yes I've thought about how best to achieve that. Could potentially put all the config in https://github.com/intercom/eslint-plugin-intercom or something else like a new eslint-plugin-intercom-recommended?